src/platforms/Token/hooks/useCollectToken.tsx

```diff
@@ -52,7 +59,17 @@ const useCollectToken = (
         method: "POST",
         body: {},
       },
-    ])
+    ]).catch((error) => {
```

Comment:
We could use the hook's `onError` here too

Reply:
This one is specifically for the claim endpoints errors. Does `useFetcherWithSign` have an `onError` function as well?

Reply:
Oh, my bad, then it's fine this way!